net-mail/mu/mu-1.4.13.ebuild

```diff
@@ -0,0 +1,91 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools elisp-common
+
+DESCRIPTION="Set of tools to deal with Maildirs, in particular, searching and indexing"
+HOMEPAGE="https://www.djcbsoftware.nl/code/mu/ https://github.com/djcb/mu"
+SRC_URI="https://github.com/djcb/mu/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="emacs guile mug"
+
+DEPEND="
+	dev-libs/glib:2
+	dev-libs/gmime:3.0
+	>=dev-libs/xapian-1.4
+	emacs? ( >=app-editors/emacs-24.4:* )
+	guile? ( >=dev-scheme/guile-2.2 )
+	mug? (
+		net-libs/webkit-gtk:4
```

Comment:
If we link against something and it has a subslot, we should [subscribe](https://devmanual.gentoo.org/general-concepts/dependencies/) to it with :=.